site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -167,6 +175,37 @@ gcloud container node-pools create windows \
 
 where [CLUSTER_NAME] is the name of the cluster you created.
 
+{{% feature publishVersion="1.29.0" %}}
+
+### Create an Autopilot mode cluster for Agones
+
+Agones supports Autopilot clusters in [alpha](/docs/guides/feature-stages/#alpha).
+
+```bash
+gcloud container clusters create-auto [CLUSTER_NAME] \
+  --region=[COMPUTE_REGION] \
+  --release-channel=[RELEASE_CHANNEL] \
+  --autoprovisioning-network-tags=game-server \
+```
+Replace the following:
+
+* CLUSTER_NAME: the name of your cluster.
+* COMPUTE_REGION: the
+  [Google Cloud region](https://cloud.google.com/compute/docs/regions-zones#available)
+  for the cluster.
+* RELEASE_CHANNEL: the
+  [GKE release channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions)
+  for the cluster. Can be one of `rapid`, `regular`, or `stable`. The default
+  is `regular`. The release channel must use version {{% k8s-version %}} or
+  later.
+
+In Autopilot clusters, GKE manages the nodes, so you won't create dedicated
```

Comment:
Where do we want to put the Autopilot specific `helm install` instructions?

Reply:
I didn't realise that they were different...hm. I'm torn because putting it here would look weird since there's a dedicated install page, but putting it on that page would mean that one cloud provider's product would have its content outside of the "create your GCP environment" guide.

What are your thoughts on it?

Reply:
I'm thinking that while we still have the feature flags, let's put the install instructions here.

Eventually, the install process will converge (features will be promoted), so we can drop it as we move forward.

@zmerlynn WDYT?

Reply:
I agree with @markmandel - let's put it here for now. It's a little gross but 🤷 , we'll figure it out as we mature the feature flags.

Reply:
Done